core/templates/dev/head/css/oppia.css

```diff
@@ -77,6 +81,10 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
 .material-icons.md-dark {
   color: rgba(0, 0, 0, 0.54);
 }
+.middle{
```

Comment:
Class name is too vague.

Not sure if you need this though. What happens if you just use md-18? I think you mentioned it looked odd; could you attach a screenshot?